src/components/resource/_fields/custom-embargo/custom-embargo-fields.js

```diff
@@ -159,4 +156,20 @@ export default class CustomEmbargoFields extends Component {
       </div>
     );
   }
+
+  renderEmbargoField = ({ fields, meta: { initial } }) => {
+    return fields.value.length
+      ? this.renderInputs(fields, initial)
```

Comment:
please move fields and initial props to an object and pass it as {...propObject} to "render" functions

Reply:
What is `initial`? I think it will be better to give it another name